### DISCUSSION TITLE
Add "solved" code?

### DISCUSSION THREAD

**Discussion by ananduri** [dendibakh/perf-ninja#79] [General]
Hello, thanks a lot for putting this material together. It looks really interesting.

I was wondering if you had plans to add code containing the "solved" versions of the lab assignments? It would be really helpful to study that directly, in addition to the summary videos.

**Comment by liangzhongkai** [comment]
您的来信我已收到，谢谢

**Comment by dendibakh** [comment]
There is a branch named "golden" that has reference solutions for all the labs. These solutions _should_ pass all tests, however, you may be able to squeeze some more gains from fine-tuning it for your particular platform.
